src/wifi/AdafruitIO_MKR1010.h

```diff
@@ -0,0 +1,50 @@
+/*!
+ * @file AdafruitIO_MKR1010.h
+ *
+ * Adafruit invests time and resources providing this open source code.
+ * Please support Adafruit and open source hardware by purchasing
+ * products from Adafruit!
+ *
+ * Copyright (c) 2015-2016 Adafruit Industries
+ * Authors: Tony DiCola, Todd Treece
+ * Licensed under the MIT license.
+ *
+ * All text above must be included in any redistribution.
+ */
+
+#ifndef ADAFRUITIO_MKR1010_H
+#define ADAFRUITIO_MKR1010_H
+
+#if defined(ARDUINO_SAMD_MKR1010)
```

Comment:
This line is not required as you already include this file in AdafruitIO_WiFi.h here: https://github.com/adafruit/Adafruit_IO_Arduino/pull/136/files#diff-11491baaaad5c5c576df8f3844aa49b8R23

Please remove it